src/clone.c

```diff
@@ -163,6 +246,11 @@ SYS_FUNC(clone3)
 
 	fetch_size = MIN(size, sizeof(arg));
 
+	const kernel_ulong_t nsflags = show_namespace
+		? (CLONE_NEWCGROUP|CLONE_NEWIPC|CLONE_NEWNS|CLONE_NEWNET|CLONE_NEWPID|CLONE_NEWTIME|CLONE_NEWUTS|CLONE_NEWUSER)
```

Comment:
unlike `clone` flags, `clone3` flags are of type uint64_t.
Also do not forget CLONE_INTO_CGROUP.

Reply:
fixed locally.